Dockerfile

```diff
@@ -29,6 +29,17 @@ RUN opam init \
     && opam install -y --deps-only . \
     && rm Hemlock.opam
 
+FROM --platform=${HEMLOCK_PLATFORM} base AS latest
+USER hemlock
+WORKDIR /home/hemlock/Hemlock
+COPY --chown=hemlock:hemlock /.git/ .git
+RUN git reset --hard \
+    && cd bootstrap \
+    && find . -type f -regex '.*\mli?' | xargs -- opam exec -- ocp-indent -i \
+    && git diff --exit-code \
+    && opam exec -- dune build \
+    && opam exec -- dune runtest
+
```

Comment:
Perhaps this would be more useful.

```
RUN git reset --hard

FROM --platform=${HEMLOCK_PLATFORM} latest AS tested
USER hemlock
WORKDIR /home/hemlock/Hemlock/bootstrap
RUN find . -type f -regex '.*\mli?' | xargs -- opam exec -- ocp-indent -i \
    && git diff --exit-code \
    && opam exec -- dune build \
    && opam exec -- dune runtest
```

Reply:
The idea being that `latest` includes a clean codebase. `tested` ensures it passes tests.

Reply:
LGTM.